src/console-services.yml

```diff
@@ -99,57 +175,125 @@
   description: 'EC2 - Virtual Servers in the Cloud'
   url: 'https://console.aws.amazon.com/ec2/home'
 
+- command: ecr
+  description: 'ECR - Fully-managed Docker container registry'
+  url: 'https://console.aws.amazon.com/ecr/home'
+
 - command: ecs
-  description: 'EC2 Container Service - Run and Manage Docker Containers'
+  description: 'ECS - Run and Manage Docker Containers'
   url: 'https://console.aws.amazon.com/ecs/home'
 
 - command: efs
-  description: 'Elastic File System - Fully Managed File System for EC2'
+  description: 'EFS - Managed File Storage for EC2'
   url: 'https://console.aws.amazon.com/efs/home'
 
+- command: eks
+  description: 'EKS - Managed Kubernetes on AWS'
+  url: 'https://console.aws.amazon.com/eks/home'
+
 - command: elasticache
   description: 'ElastiCache - In-Memory Cache'
   url: 'https://console.aws.amazon.com/elasticache/home'
 
-- command: elasticbeanstalk
-  description: 'Elastic Beanstalk - Run and Manage Web Apps'
-  url: 'https://console.aws.amazon.com/elasticbeanstalk/home'
```

Comment:
why was this removed? this is actually one of the few dashboards i still use at my new job, haha

Reply:
ooops! My bad. Putting it back. Sorry.